client/package.json

```diff
@@ -37,6 +37,7 @@
     "gl-mat4": "^1.2.0",
     "gl-matrix": "^3.3.0",
     "gl-vec3": "^1.1.3",
+    "history": "^4.10.1",
```

Comment:
Add `history` in deps, because we're explicitly using it in `client/src/util/history.js` like so:

`import { createBrowserHistory } from "history";`